### ISSUE TITLE
Adding a Reset backend Search button or (X)

### ISSUE THREAD

**Issue by the-specialist2005** [octobercms/october#3425]
New Suggestion to add a button to clear search field in the back-end instead of having to delete the text letter by letter in order to clear search filtration.

**Comment by LukeTowers** [comment]
@the-specialist2005 good idea, I'd welcome that; it would also serve as a solution for an issue that occurs when the search is saved in the user's session / cookies but doesn't get populated in the search box so the records are filtered by an invisible search. Make a PR and then ping me on it to review it

**Comment by vosco88** [comment]
@LukeTowers I've started to work on this one. So far I've put the button to the left of the search like below as not to have the input field too crowded when closed.
![image](https://user-images.githubusercontent.com/14372933/49343138-78e16500-f664-11e8-9124-95453c1bd9f9.png)
Other placement I was thinking is to replace the search icon with the 'X' icon but I did not want to loose the search icon. So wanted to check with you what would actually be preferable.
